tests/torch/pruning/helpers.py

```diff
@@ -620,6 +620,26 @@ def forward(self, x):
         return y1, y2
 
 
+class SplitIdentityModel(nn.Module):
+    #         (input)
+    #            |
+    #         (conv1)
+    #            |
+    #         (chunk)
+    #            |
+    #         (conv2)
+    def __init__(self):
+        super().__init__()
+        self.conv1 = create_conv(1, 4, 1, 1)
+        self.conv2 = create_conv(4, 8, 1, 1)
+
+    def forward(self, x):
+        x = self.conv1(x)
+        y1 = torch.chunk(x, chunks=1, dim=1)
+        y1 = self.conv2(y1[0])
```

Comment:
Now I know what case you're talking about. There are other cases with dim=1, but there are no cases with more than one as shown in this. But does this kind of case actually exist? Then, is there a model that uses chunk/split? However, it looks good to add it for now.

Reply:
We caught identity concat in one of OTE model, so identity chunk is possible :)

Reply:
I see. Thank you for checking this part in detail.